adafruit_circuitplayground/circuit_playground_base.py

```diff
@@ -184,6 +184,58 @@ def detect_taps(self, value):
                     value, 60, time_limit=10, time_latency=50, time_window=255
                 )
 
+    # pylint: disable-msg=too-many-arguments
+    # Many default arguments which will probably not need changing in most cases
+    def configure_taps(
```

Comment:
You can `disable` Pylint for a specific line, which removed the need to `enable` it again later. See the suggestion for the way to use the `pylint: disable=` on the specific line.

Please also remove the comment as it's not necessary.
```suggestion
    def configure_taps(  # pylint: disable-msg=too-many-arguments
```

Reply:
Changed.